<Simulation color="True" verbosity="debug">
  <RunInfo>
    <WorkingDir>run</WorkingDir>
    <Sequence>sample</Sequence>
    <batchSize>1</batchSize>
  </RunInfo>
  <TestInfo>
    <name>framework/SingleRuns/externalModelFails</name>
    <author>talbpw</author>
    <created>2017-01-27</created>
    <classesTested>Steps.SingleRun,Model.ExternalModel</classesTested>
    <description>
       This test assures the SingleRun produces the correct error when paired with the External model.
    </description>
  </TestInfo>

  <Steps>
    <SingleRun name="sample">
      <Input class="DataObjects" type="PointSet">dummyIn</Input>
      <Model class="Models" type="ExternalModel">projectile</Model>
      <Output class="DataObjects" type="PointSet">sample</Output>
    </SingleRun>
  </Steps>

  <Models>
    <ExternalModel ModuleToLoad="../../AnalyticModels/projectile" name="projectile" subType="">
      <variables>v,ang,r</variables>
    </ExternalModel>
  </Models>

  <DataObjects>
    <PointSet name="dummyIn">
      <Input>v,ang</Input>
      <Output>OutputPlaceHolder</Output>
    </PointSet>
    <PointSet name="sample">
      <Input>v,ang</Input>
      <Output>r</Output>
    </PointSet>
  </DataObjects>

  <OutStreams>
    <Print name="sample">
      <type>csv</type>
      <source>sample</source>
    </Print>
  </OutStreams>
</Simulation>
